mrblib/mitamae/inline_backends/file_backend.rb

```diff
@@ -19,6 +19,22 @@ def get_file_mode(path)
       rescue => e
         Specinfra::CommandResult.new(stdout: '', stderr: e.message, exit_status: 1)
       end
+
+      def get_file_owner_user(path)
+        uid = File.stat(path).uid
+        passwd = Etc.getpwuid(uid)
+        Specinfra::CommandResult.new(stdout: passwd.name, stderr: '', exit_status: 0)
+      rescue => e
+        Specinfra::CommandResult.new(stdout: '', stderr: e.message, exit_status: 1)
```

Comment:
By the way, I think InlineBackend is not related to Specinfra and it should have a different struct like Specinfra::CommandResult.  WDYT?